### PR TITLE
HS-399315 - Dynamic Years Dropdown

### PR DESCRIPTION
**Pull request by wrandall22**
This PR updates the dropdown to be dynamic such that we don't have to manually add the next year every new year for paycheck history.

[Helpscout](https://secure.helpscout.net/conversation/1046250961/399315?folderId=381778)

### CODE REVIEW

File: js/ng-app-paycheck-history/controllers/dashboard.js [hunk 1]
```diff
@@ -45,6 +45,15 @@
...
 
+      scope.getYears = function() {
+        var totalYears = new Date().getFullYear() - 2012;
+        var years = [];
+        for (var i = 0; i < totalYears; i++) {
+            years.unshift(2012 + i);
+        }
+        return years;
+      }
+
```
Comment (mattdrees) [inline js/ng-app-paycheck-history/controllers/dashboard.js]: this will go up to 2019, right? I think probably we want it to go to 2020.

**Comment by wrandall22** [comment]
That's what I wasn't sure about. I thought maybe the default was the current year and the dropdown was for past years. Before now, it only went to 2018.

**Comment by mattdrees** [comment]
I think that's only because we forgot to update the code last year. 

Same thing happened two years ago - see  https://github.com/CruGlobal/paycheck-history/pull/13/commits/6d7701f300633481e7dc6d7ace173c8994b1c1eb.

The behavior of the site only breaks badly enough the second year it is neglected. The first year after, the degradation isn't enough for people to complain, I guess.

**Comment by wrandall22** [comment]
I see. I'll update it to include the current year then.

**Comment by mattdrees** [comment]
thanks Bill!

**Pull request merged**